pages/pages.json

```diff
@@ -1,117 +1,34 @@
 {
   "pages": {
-    "getting-started": {
-      "path": "/getting-started",
-      "theme": "theme",
-      "extensible": true
-    },
-    "getting-started/nested-a": {
-      "path": "/getting-started/nested-a"
-    },
-    "getting-started/nested-b": {
-      "path": "/getting-started/nested-b"
-    },
-    "getting-started/nested-param": {
-      "path": "/getting-started/:slug/nested-param"
-    },
-    "getting-started/nested-param/deeply-nested": {
-      "path": "/getting-started/:slug/nested-param/:deep/deeply-nested"
-    },
-    "getting-started/container": {
-      "path": "/getting-started/container"
-    },
-    "legacy-getting-started": {
-      "path": "/legacy-extensions/render-example",
+    "carousel": {
+      "path": "/",
       "theme": "theme"
-    },
-    "legacy-without-tachyons": {
-      "path": "/legacy-extensions/render-example-without-tachyons",
-      "theme": "theme",
-      "disableExternals": ["tachyons"]
     }
   },
+
   "extensions": {
-    "getting-started": {
-      "component": "index",
-      "props": {
-        "page": "getting-started/nested-a"
-      }
-    },
-    "getting-started/nested-a": {
-      "component": "nested",
-      "props": {
-        "page": "getting-started/nested-b"
-      }
-    },
-    "getting-started/nested-b": {
-      "component": "nested",
-      "props": {
-        "page": "getting-started/nested-param",
-        "targetParams": {
-          "slug": "myslug"
-        }
-      }
-    },
-    "getting-started/nested-param": {
-      "component": "nested",
-      "props": {
-        "page": "getting-started/nested-param/deeply-nested",
-        "targetParams": {
-          "slug": "myslug",
-          "deep": "second"
+    "carousel": {
+      "component": "Carousel",
+      "props": {
+        "autoplay": false,
+        "arrowColor": "FFFFFF",
+        "banner1": {
+          "image": "http://dicasdacapital.com.br/wp-content/uploads/2018/02/economia-creme-ameixa-negra-boticario-20180216-001.jpeg",
```

Comment:
Remove the default image. When a user installs the banner on your store, he must set the images.